schema/v1.1.0/dataset_config_models_extended.py

```diff
@@ -84,6 +84,7 @@
 
 logger = logging.getLogger("dataset_config_validate")
 
+FIELD_EXCLUDELIST = {}
```

Comment:
If a variable is named in all caps, we should treat it as a constant. 

Also, what is your thought on renaming this to `VALIDATION_EXCLUSIONS`?

Reply:
Sure, I'll go ahead and change this to `validation_exclusions`